Tasks/Common/webdeployment-common/WebConfigTemplates/go

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated web.config for Microsoft Azure. Remove this comment to prevent
+     modifications being overwritten when publishing the project.
+-->
+<configuration>
+    <system.webServer>
+        <handlers>
+            <add name="httpplatformhandler" path="*" verb="*" modules="httpPlatformHandler" resourceType="Unspecified" />
+        </handlers>
+        <httpPlatform processPath="D:\home\site\wwwroot\{GoExeFileName}" startupTimeLimit="60">
```

Comment:
This is specific to Azure Web apps. Don't have this default in common. Pass it from AzureRM task. This common helper can be used from other places as well.

Also can the processPath be a relative path?

Reply:
Process path cannot be relative.
And i have removed it from template file and moved it to process parameter